src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileParser.cs

```diff
@@ -75,87 +74,151 @@ private XmlReader ApplyIgnorables(string file)
 			var document = new XmlDocument();
 			document.Load(file);
 
-			var ignorables = FindIgnorables(document);
+			var (ignorables, shouldCreateIgnorable) = FindIgnorables(document);
+			var conditionals = FindConditionals(document);
 
-			if (ignorables != null)
+			shouldCreateIgnorable |= conditionals.ExcludedConditionals.Count > 0;
+
+			var adjusted = File.ReadAllText(file, Encoding.UTF8);
+			var hasxBind = adjusted.Contains("{x:Bind");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 'adjusted.Contains' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)